### DISCUSSION TITLE
Is prisma logs the transaction details in mongoDB atlas oplog ?

### DISCUSSION THREAD

**Discussion by gbharath5182** [prisma/prisma#22344] [Q&A]
### Question

Is prisma logs the transaction details in mongoDB atlas oplog ? When I am using AWS DMS for migrating data from MongoDB to SQL Server for replicate ongoing changes, I am getting below error.
```
**Error Message  ...**
Last Error Error reading the database log. Stop Reason FATAL_ERROR Error Level FATAL
**Cloud watch log**
"Unable to find transaction in opLog. (change_streams_capture.c:1041)"
```

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by ludralph** [comment]
Hi @gbharath5182 👋 

Thank you for raising this question.

Prisma does not directly log transactions in MongoDB's oplog. Prisma is a database ORM (Object Relational Mapper) that sits on top of your database and provides an abstraction layer for interacting with it. While it can be used with MongoDB, it doesn't interact directly with the oplog. Prisma does provide its own mechanism for [logging](https://www.prisma.io/docs/orm/prisma-client/observability-and-logging/logging).

If you would want to see this feature available within Prisma, you could open a [feature request](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) and document your use case so our engineers can have a look.

If this answers your question, it would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 

Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏

**Reply by gbharath5182** [reply]
Hi @ludralph 
Sorry for late reply... Thank you for addressing my issue.
How can I achieve AWS replicate ongoing migration ? any alternate solution please..!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
